src/index.html

```diff
@@ -1,13 +1,420 @@
+            <div class="explore-online-block__wrapp explore-online-block__wrapp--square">
+              <img
+                class="explore-online-block__img"
+                src="./images/explore-online-left.png"
+                alt="explore-online"
+              >
+            </div>
+            <div class="explore-online-block__bottom">
+              <h3 class="explore-online-block__title">Collection Areas</h3>
+              <p class="explore-online-block__description">
+                These collection areas study, exhibit, and care for the objects in The Met collection.
+              </p>
+            </div>
+          </article>
+          <article class="explore-online-block">
+            <div class="explore-online-block__wrapp explore-online-block__wrapp--square">
+              <img
+                class="explore-online-block__img"
+                src="./images/explore-online-center.png"
+                alt="explore-online"
+              >
+            </div>
+            <div class="explore-online-block__bottom">
+              <h3 class="explore-online-block__title">Travel</h3>
+              <p class="explore-online-block__description">
+                Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.
+              </p>
+            </div>
+          </article>
+          <article class="explore-online-block">
+            <div class="explore-online-block__wrapp explore-online-block__wrapp--square">
+              <img
+                class="explore-online-block__img"
+                src="./images/explore-online-right.png"
+                alt="explore-online"
+              >
+            </div>
+            <div class="explore-online-block__bottom">
+              <h3 class="explore-online-block__title">Virtual group tours</h3>
+              <p class="explore-online-block__description">
+                Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.
+              </p>
+            </div>
+          </article>
+        </div>
+      </section>
+
+      <section class="contact-us" id="contact-us">
+        <h2 class="section-title contact-us__title">Contact us</h2>
+        <div class="contact-us-content">
+          <div class="contact-us-content__fifth-avenue">
+            <h3
+              class="contact-us-content__title contact-us-content__text-stl-1">
+              The Met Fifth Avenue
+            </h3>
+            <p
+              class="contact-us-content__phone contact-us-content__text-stl-2"
+            >
+              Phone
+            </p>
+            <a
+              href="tel:+12125357710"
+              class="contact-us-content__phone-number contact-us-content__text-stl-1">
+              +1 212-535-7710
+            </a>
+            <p
+              class="contact-us-content__address contact-us-content__text-stl-2"
+            >
```

Comment:
If we have only 1 or 2 attributes, then there is no need to move them to a new line, this should be done only when there are 3 or more attributes
```suggestion
            <p class="contact-us-content__address contact-us-content__text-stl-2">
```

Reply:
Clear, thank you.